src/main/java/com/minecolonies/coremod/sounds/ModSoundEvents.java

```diff
@@ -17,6 +17,13 @@
      * List of sound handlers.
      */
     public static final List<AbstractWorkerSounds> handlers = new ArrayList<>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this member "protected". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2386)